include/element.cc

```diff
@@ -0,0 +1,48 @@
+
+
+//! Calculate volume of an element
+//! \tparam Tdim Dimension of the mesh
+//! \tparam Tvertices Number of vertices in an element
+//! \param[in] 3x8 Matrix of element vertices coordinates
```

Comment:
There are no input parameters

Reply:
`\retval` for return

Reply:
do you want to make it `.tcc` btw? or want to keep it as `cc`?

Reply:
This is not a templatised class, so it won't be a `.tcc` file.